### PR TITLE
Issue 1690: Pravega.io doc updates

### PR DESCRIPTION
**Pull request by askewj**
**Change log description**
- Updated pravega.io documents

**Purpose of the change**
Updated pravega.io documents

**What the code does**
Doc changes.

**How to verify it**
Look at pravega.io
Deploy PravegaGroups.json via Marathon

### CODE REVIEW

File: PravegaGroup.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
     {
       "id": "/pravega/bkgroup",
       "apps": [
-      { 
-          "id": "/pravega/bkgroup/bookie", 
+      {
```
Comment (fpj) [inline PravegaGroup.json]: Could you submit these pravega group changes in a separate PR, please? They are unrelated to doc changes.

File: docs/roadmap.md [hunk 2]
```diff
@@ -9,97 +9,49 @@ You may obtain a copy of the License at
...
+Security for this release will focus on support for securing Pravega external interfaces along with basic access controls on stream access and administration.
+-  Access Control on Stream operations
+-  Auth between Clients and Controller/SegmentStore
+-  Auth between SegmentStore and Tier 2 Storage
+
+## Pravega Connectors
+Pravega ecosystem interconnectivity will be augmented with the following:
+-  Expanded Flink connector support (batch & table API support)
+-  Logstash connector
+-  Other (TBD)
```
Comment (fpj) [inline docs/roadmap.md]: TBD sounds like we forgot to update. Should we say something like others under development? Perhaps we should also add a note inviting others to contribute, like "interested in writing a connector? talk to us...".

File: PravegaGroup.json (hunk 1 above)
Comment (askewj) [inline PravegaGroup.json]: Sure, reverted it in this PR.

File: docs/roadmap.md (hunk 2 above)
Comment (askewj) [inline docs/roadmap.md]: done
